fhttp/httprunner.go

```diff
@@ -154,6 +158,7 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 		total.sizes.Transfer(httpstate[i].sizes)
 		total.headerSizes.Transfer(httpstate[i].headerSizes)
 	}
+	total.SentRequestSizeBPS = float64(total.SentRequestSize) / total.ActualDuration.Seconds()
```

Comment:
why is that needed to be persisted ? also BPS is ambigous

Reply:
At the #185 task you told me to collect requestSize and in terms of kbps.

> we should add send size and kbytes/sec calc

BPS was my fault. I had to make it kbps instead of bps. But if you are saying that we shouldnt persist the Kbps value as well, it is ok for me.

Reply:
Sure, I can make it like that.